Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -138,59 +141,36 @@ - (BOOL)togglePause:(BOOL)pause forCall:(OCTCall *)call error:(NSError **)error
             return NO;
         }
 
-        [self.calls updateCall:call updateBlock:^(OCTCall *callToUpdate) {
-            callToUpdate.status = pause ? OCTCallStatusPaused : OCTCallStatusActive;
-        }];
-
+        switch (control) {
+            case OCTToxAVCallControlResume:
+                [self.calls updateCall:call updateBlock:^(OCTCall *callToUpdate) {
+                [callToUpdate startTimer];
+                callToUpdate.status = OCTCallStatusActive;
+            }];
```

Comment:
Uncrustify has some problems with indentation inide blocks. Can you try to fix it manually (uncrustify may ignore it, well, it does that sometimes).